index.js

```diff
@@ -0,0 +1,28 @@
+const calculate = (calculatorInputs) => {
+  const acceptedInputs = calculatorInputs.filter((input) => {
+    return input || input === 0 || input === null
+  })
+  const [a, operator, b] = acceptedInputs
+
+  for (let i = 0; i < acceptedInputs.length; i++) {
+    if (operator === '*') {
+      return +a * +b
+    }
+
+    if (operator === '+') {
+      return +a + +b
+    }
+
+    if (operator === '-') {
+      return +a - +b
+    }
+
+    if (operator === '/') {
+      return +a / +b
+    }
+  }
+
+  return NaN
+}
```

Comment:
This should be a switch and not in a loop. You popped your variables off the array on line 5, no need to loop after that.